joinfaces-autoconfigure/src/main/java/org/joinfaces/autoconfigure/weld/package-info.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2016-2016 the original author or authors.
```

Comment:
Please adjust the year here.

Reply:
Like this Lars?
Copyright 2016-2018 the original author or authors.

Reply:
Done